cmake/TaichiCore.cmake

```diff
@@ -34,7 +34,7 @@ endif()
 
 option(USE_STDCPP "Use -stdlib=libc++" OFF)
 option(TI_WITH_CUDA "Build with the CUDA backend" OFF)
-option(TI_WITH_OPENGL "Build with the OpenGL backend" OFF)
+option(TI_WITH_OPENGL "Build with the OpenGL backend" ON)
```

Comment:
Maybe we should keep it `OFF`?

Reply:
Can I make GL default to ON in something ignored by `.gitignore`? It's not easy to keep in mind to specify that long argument.